src/coreclr/src/jit/emitxarch.cpp

```diff
@@ -13985,6 +13985,7 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
 
     switch (ins)
     {
+        case INS_align:
```

Comment:
What is this change?

Reply:
When we set `COMPlus_JitAlignLoops=1` and try to print the disassembly, an entry of `align` instruction was missing and we were hitting an assert when calculating the perfscore.